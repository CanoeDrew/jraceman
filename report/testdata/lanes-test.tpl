This is the lanes-test report.
{{include "org.jimmc.jraceman.Lanes" -}}
