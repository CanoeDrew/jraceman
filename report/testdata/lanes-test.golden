This is the lanes-test report.
This is a stub for the Lanes report.
